prqlc/prqlc/Cargo.toml

```diff
@@ -22,12 +22,14 @@ cli = [
   "color-eyre",
   "colorchoice-clap",
   "is-terminal",
+  "lsp",
   "minijinja",
   "notify",
   "serde_yaml",
   "walkdir",
 ]
 default = ["cli"]
+lsp = ["lsp-server", "lsp-types"]
```

Comment:
Can we add a note that this is just a stub at the moment; doesn't add functionality yet?